### PR TITLE
Add changelog for removing beta UI, remove beta UI known issues

### PR DESCRIPTION
**Pull request by bheftel**
- This adds an entry to the changelog for removing the beta UI
- Removes beta UI issues from Known Issues page

### CODE REVIEW

File: content/changelog.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ weight: 900
 toc: true
 ---
 
+## Sept 21, 2023
+
+### What's New
+ 
+- {{% icon-feature %}} **Removal of NGINX Amplify Beta User Interface**
+
+  The NGINX Amplify Beta User Interface has been removed. We’re choosing to go in a different direction. Thank you to everyone who tried out the new UI or provided feedback.
```
Comment (bheftel) [inline content/changelog.md:14]: @travisamartin - please feel free to change this. We weren't sure how to balance 
the technical tone of a changelog with seeming authentic in thanking people for giving feedback.

Comment (travisamartin) [inline content/changelog.md:14]:
```suggestion
  The NGINX Amplify Beta User Interface is no longer available. We're shifting our focus to concentrate on core initiatives. A big thank you to all who tested the interface or shared their thoughts.
```

Comment (travisamartin) [reply content/changelog.md:14]: Nice. The thank you at the end adds a touch of gratitude and appreciation for our users, which goes a long way.

I expanded on the reason for the pivot without giving away too much detail. Transparency is good.

**Review by travisamartin — APPROVED** [review]

File: content/changelog.md (hunk 1 above)
Comment (bheftel) [reply content/changelog.md:14]: Thanks Travis! I agree, we should be as transparent as we can get away with.

**Review by achawla2012 — APPROVED** [review]

**Pull request merged**